eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -2,15 +2,23 @@ jobs:
 - job: BuildTest
   strategy:
     matrix:
-      Linux:
+      Linux_x64:
         vm.image: 'ubuntu-18.04'
         vcpkg.deps: 'curl[ssl]'
```

Comment:
this could become annoying when running tests because unlike curl[winssl] curl with openssl may not use the "default" cert store, as far as there is a default on linux. Just something to keep in mind for the future

Reply:
Issue #220